include/freerdp/settings.h

```diff
@@ -1562,6 +1562,9 @@ struct rdp_settings
 	ALIGN64 char* ActionScript;
 	ALIGN64 DWORD Floatbar;
 	ALIGN64 char* XSelectionAtom;
+
+	ALIGN64 char* VAAPIConnectionType;
```

Comment:
Unify to a single string, let the backend interpret it.
Also add it to public API